### PR TITLE
feat(package): add cli tooling

### PR DESCRIPTION
**Pull request by InteractiveTimmy**
# **DRAFT -- DO NOT MERGE!**

# Description

The scope of the changes in this pull request are to set up the foundations for core JS SDK configuration packages for modern implementation. This also includes the mounting of a new `@webex/cli-tools` package that will be consumed when setting up deploy workflows in CI/CD.

### CODE REVIEW

**Review by chrisadubois — COMMENTED** [review]
my only prob with this PR is that jasmine is now in conjunction with mocha/chai/jest

else than that, looks good and mostly cleans the configs up and structures them into packages

File: packages/config/eslint/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const config = {
+  root: true,
+  env: {
+    node: true,
+  },
+  extends: ['@webex/eslint-config/core'],
```
Comment (arun3528) [inline packages/config/eslint/.eslintrc.js:6]: this will include all the default rules we had and additional rules

Comment (arun3528) [reply packages/config/eslint/.eslintrc.js:6]: what is the difference between the legacy eslint and the new eslint package ?

File: packages/tools/cli/api-extractor.config.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const build = require('@webex/api-extractor-config');
+
+build();
```
Comment (arun3528) [inline packages/tools/cli/api-extractor.config.js:3]: do we still need build or can we just use the cli command ?

**Review by arun3528 — APPROVED** [review]
thanks tim

**Pull request merged**